src/coreclr/jit/fgbasic.cpp

```diff
@@ -1196,6 +1196,7 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, Fixed
                             case NI_X86Base_X64_BitScanForward:
                             case NI_X86Base_BitScanReverse:
                             case NI_X86Base_X64_BitScanReverse:
+                            case NI_Vector512_Create:
```

Comment:
nit: Would be nice to keep this grouped with the other `NI_Vector###_Create` methods.

Reply:
Done